src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/attachment.yml

```diff
@@ -0,0 +1,69 @@
+admin_attachments_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Attachment:index
+    _legacy_controller: AdminAttachments
+#    _legacy_link: AdminAttachments
```

Comment:
Commented?

Reply:
probably because the page is not visible yet, so the legacy links are not updated.